src/consensus/vote.cpp

```diff
@@ -105,8 +120,15 @@ VoteManager::VoteManager(addr_t node_addr, std::shared_ptr<DbStorage> db, std::s
   {
     auto verified_votes = db_->getVerifiedVotes();
```

Comment:
I remember that retrieve all verified votes from DB that will take a long time.
I don't know if RocksDB support get number of items. Emm... that should not work. Because in the meantime, also write votes into verified votes DB column.  And I think DB writing items are not by order guarantee